assignments/prototype-refactor.js

```diff
@@ -7,3 +7,133 @@ Prototype Refactor
 2. Your goal is to refactor all of this code to use ES6 Classes. The console.log() statements should still return what is expected of them.
 
 */
+
+
+class GameObject {
+	constructor(thing) {
+	this.createdAt = thing.createdAt;
+	this.name = thing.name;
+	this.dimensions = thing.dimensions;
+		}
```

Comment:
This would be a wee bit more readable if the blocks were properly indented. I told you about prettier and such before, right?